libs/community/langchain_community/callbacks/apigateway_callback.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Callable, Dict, List
+
+from boto3 import Session
+from langchain_core.callbacks.base import BaseCallbackHandler
+
+if TYPE_CHECKING:
+    from langchain_core.messages import BaseMessage
+    from langchain_core.outputs import LLMResult
+
+
+class StreamingAPIGatewayWebSocketCallbackHandler(BaseCallbackHandler):
```

Comment:
Can we put AWS in the name to make it clear it's AWS specific

Reply:
Sure. Since there is already an `AmazonAPIGateway` llm, maybe `StreamingAmazonAPIGatewayWebSocketCallbackHandler` is a better name.